README.md

```diff
@@ -48,3 +48,51 @@ The following types should be used as applicable `rel` types in the
 | Type                | Description |
 | ------------------- | ----------- |
 | fancy-rel-type      | This link points to a fancy resource. |
+
+## Contributing
+
+All contributions are subject to the
+[STAC Specification Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
+For contributions, please follow the
+[STAC specification contributing guide](https://github.com/radiantearth/stac-spec/CONTRIBUTING.md) Instructions
```

Comment:
The link is broken.

Reply:
It's because it's an artisanally crafted, hand-made link 🤦‍♂️

Reply:
2fcde6b